src/js/edu-benefits/5490/config/form.js

```diff
@@ -437,18 +440,6 @@ const formConfig = {
                   }
                 },
                 dateRange: uiSchemaDateRange(),
```

Comment:
Same here--comment that dateRange will disappear altogether once schema change is in.